open-company/open-company-web
# Post move

Card: https://trello.com/c/GpGdZQy8

We were missing the capability of changing section while editing a published post (ie in the fullscreen post view). This is only for desktop since on mobile the edit is always done in the entry-edit component that already has this feature.

Related PR:
- https://github.com/open-company/open-company-storage/pull/116

Also address this:
> Click X in fullscreen edit should bring back to board/AP if the user started editing directly from it

Test 1:
- go to grid view
- click on a post
- start editing it
- change the section to another existing section
- save
- [x] are you redirected to the picked section? Good (this is needed because the fullscreen post url is not valid anymore since the section changed)
- [x] do you see the post? Good

Test 2:
- go to grid view
- click on ellipses menu
- click edit
- edit the post (don't change the section)
- save or click the X
- [x] are you sent back to the view you came from? Good
- [x] do you see the post? Good

Test 3:
- go to grid view
- click on ellipses menu
- click edit
- edit the post and change the section
- save the changes
- [x] are you redirected to the section you picked? Good
- [x] do you see the post? Good

Test 4:
- go to grid view
- click on a post
- now start editing from the ellipses menu
- edit the post W/O changing the section
- save the changes
- [ ] are you NOT redirected to the section you picked? Good

Test 5 (when backend is ready):
- go to grid view
- click on a post
- start editing from the ellipses menu
- click on the chevron besides the section name
- click add a new section
- enter the data for a new section
- save the post
- [x] are you redirected to the newly created section? Good
- [x] do you see the new post? Good

### Now merge
